pyproject.toml

```diff
@@ -76,10 +76,11 @@ docs = [
 notebook-dependencies = [
     "circuit-knitting-toolbox[cplex,pyscf]",
     "quantum-serverless>=0.0.7",
+    "qiskit-nature<0.6.0",
+    "qiskit-ibm-provider",
```

Comment:
I haven't look carefully enough at this yet to understand why it is necessary, but we should put an explicit minimum version here, just like the others.

Reply:
I think it should be removed. I use it in the tutorials, but we should maybe switch to the internal "ExactSampler` in the tutorials when it's ready